src/App.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import Layout from "components/layout/Layout";
+import HomeContainer from "containers/Home";
```

Comment:
Please, rename this import. There is already an element with the name 'HomeContainer'. It's a bit confusing.